cmd/vic-machine/create/create.go

```diff
@@ -1292,61 +1312,80 @@ func logArguments(cliContext *cli.Context) {
 				continue
 			}
 			url.User = nil
-			log.Debugf("--target=%s", url.String())
+			flag := fmt.Sprintf("--target=%s", url.String())
+			log.Debug(flag)
+			args = append(args, flag)
```

Comment:
We only persist options that the user explicitly supplied during `vic-machine create`, so this should not be a problem.